### ISSUE TITLE
Crack propagation on fatigue tests

### ISSUE THREAD

**Issue by mpaes15** [dicengine/dice#276]
Hi there, Iam trying to use DICe for crack propagation study on fatigue tests. Iam using both versions 2 and 3 because I have added a live point is the crack tip but I have a couple of doubts as below:
- When I plot live points, the y axle as displacement x, is it the number of subsets? 
- How can I add live point in version 3?

![Crack](https://user-images.githubusercontent.com/117855228/201233072-147d1a61-adc1-4234-aa66-f1ed662440f2.jpg)

**Comment by dicengine** [comment]
Can you rephrase your first bullet? I'm not sure I understand the question.
You can add live points by simply right (or middle) clicking in the image in Version 3 of DICe, depending on how your mouse is mapped.

**Comment by mpaes15** [comment]
So, in the first bullet, I mean, what is the unit of "Displacement X"? Pixels, mm? Picture below clarify that.
I suppose that the numbers "-2", "-2.5" in the "y" axle is related to number of "subset size" in pixels as I defined, e.g., 33 pixels. After that, as I know in my computer 1 pixel is equal to 0.007mm, I convert from pixel to mm.
For exemple:
**Displacement X equal to 2**
- 2 x 33 pixels = 66 pixels. 
- 66 pixels x 0.007mm = 0.462mm

Can I conclude that the point "-2" in axle "Displacement X," means that the point I defined deslocated by 0.462mm?

![Deslocamento pto 1 por frame](https://user-images.githubusercontent.com/117855228/201372395-c35f5944-f139-4637-b7cd-5a7da4d62c42.jpg)

**Comment by dicengine** [comment]
The displacement in the plot is in terms of pixels. I don't know how you calibrated your system so I can't comment on what it means in terms of physical displacement, but the units of the plot are in pixels. The subset size is not involved in what's plotted.
